cptv-upload.py

```diff
@@ -12,12 +14,21 @@ def main():
     parser.add_argument("password", help="Password to authenticate with")
     parser.add_argument("groupname", help="Group name to upload on behalf of")
     parser.add_argument("devicename", help="Device to upload on behalf of")
-    parser.add_argument("filename", help="File to upload")
+    parser.add_argument("filename", help="File to upload. If it is a directory will upload all cptv files in that dir")
+
     args = parser.parse_args()
 
     api = API(args.server_url, args.username, args.password)
-    api.upload_recording(args.groupname, args.devicename, args.filename)
-
+    if os.path.isdir(args.filename):
+        print("uploading multiple files")
```

Comment:
there's probably no need for this nor the "uploading just one recording" message, especially given that the name of each file is printed for the directory case